app/inbox/views.py

```diff
@@ -0,0 +1,127 @@
+# -*- coding: utf-8 -*-
+"""Define view for the inbox app.
+
+Copyright (C) 2018 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+
+import json
+
+from django.contrib.auth.decorators import login_required
+from django.contrib.staticfiles.templatetags.staticfiles import static
+from django.core.paginator import Paginator
+from django.http import HttpResponse, JsonResponse
+from django.template.response import TemplateResponse
+from django.utils.translation import gettext_lazy as _
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_GET, require_http_methods
+
+from inbox.models import Notification
+
+
+@login_required
+@require_GET
+def notifications(request):
+    """Handle all notifications."""
+    limit = int(request.GET.get('limit', 10))
+    page = int(request.GET.get('page', 1))
```

Comment:
is there a way i can generate a bunch of fake notifications? i d like to test the pagination

Reply:
yes in the admin you can generate it

Reply:
Fixtures? 🤔

Reply:
ups sorry not a bunch, just one by one